tests/backup_restore_test.go

```diff
@@ -22,15 +21,18 @@ func TestServer_BackupAndRestore(t *testing.T) {
 	config.Monitor.StoreEnabled = true
 	config.Monitor.StoreInterval = toml.Duration(time.Second)
 
-	fullBackupDir, _ := ioutil.TempDir("", "backup")
+	fullBackupDir, _ := os.MkdirTemp("", "backup")
 	defer os.RemoveAll(fullBackupDir)
 
-	partialBackupDir, _ := ioutil.TempDir("", "backup")
+	partialBackupDir, _ := os.MkdirTemp("", "backup")
 	defer os.RemoveAll(partialBackupDir)
 
-	portableBackupDir, _ := ioutil.TempDir("", "backup")
+	portableBackupDir, _ := os.MkdirTemp("", "backup")
 	defer os.RemoveAll(portableBackupDir)
 
+	shardBackupDir, _ := os.MkdirTemp("", "backup")
+	defer os.RemoveAll(shardBackupDir)
```

Comment:
Why were these lines introduced?

Reply:
Oh, I missed this comment. Oh, it looks like I accidentally added this when I cherry-picked the commit. I've removed these lines in a new commit.